packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+/* @conditional-compile-remove(video-background-effects) */
+import { Panel } from '@fluentui/react';
+
+/**
+ * Pane that is used to show video effects button
+ * @private
+ */
+/** @beta */
+export const VideoEffectsPane = (props: {
+  showPanel: boolean;
+  setShowPanel: (showPane: boolean) => void;
+}): JSX.Element => {
+  return VideoEffectsPaneTrampoline(props.showPanel, props.setShowPanel);
+};
+
+const VideoEffectsPaneTrampoline = (showPanel: boolean, setShowPanel: (showPane: boolean) => void): JSX.Element => {
+  /* @conditional-compile-remove(video-background-effects) */
+  return (
+    <Panel
```

Comment:
if the panel component needs this state could we maybe just make this a stack and control state in the composite? or is there some desired behaviour of the component that makes us want to use it?

Reply:
I agree with Don here. We should store the state in the composite and show this component based on that state. Is there is a reason for storing panel state in the panel component itself?

Reply:
Yeah we are storing the panel state in Composite and the open/close of panel is controlled by composite state